airflow/config_templates/config.yml

```diff
@@ -662,6 +663,7 @@ logging:
         log files will be deleted after they are uploaded to remote location.
       version_added: 2.6.0
       type: string
+      sensitive: true
```

Comment:
Does this one need to be sensitive too?

Reply:
it's essentially same as backend_kwargs for secrets. should people pass sensitive info there? probably not. could they, since it's an arbitrary custom backend -- sure. same applies here i think. i'm not sure any of our built-in task handlers could be / would be used in this way, but i would have said same about backend.